src/sentry/migrations/0160_add_organizationmember_to_externaluser.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 1.11.29 on 2021-02-10 00:44
+
+from django.db import migrations
+import django.db.models.deletion
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0159_create_externaluser_table"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="externaluser",
+            name="organizationmember",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                default=None,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="sentry.OrganizationMember",
+            ),
+            preserve_default=False,
+        ),
+        migrations.RemoveField(
+            model_name="externaluser",
+            name="user",
+        ),
+        migrations.AlterUniqueTogether(
+            name="externaluser",
+            unique_together=set([("organizationmember", "provider", "external_id")]),
+        ),
```

Comment:
I think this needs to happen before `RemoveField`, not sure if you modified the migration at all or if this is a bug in the migration generator.

Reply:
@wedamija no this is the generated migration. I'll update it.